### PR TITLE
Update @audius/sdk to 2.0.3-beta.4

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Updates @audius/sdk in all relevant packages to consume some DMs changes.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Review by dylanjeffers — APPROVED** [review]
great! can we wait until the sdk work is merged so we get v3 with that, then we get v4 with yours?

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Pull request merged**
